src/components/Tabs.js

```diff
@@ -8,7 +8,9 @@ const CustomTabs = props => (
         <Tab key={`tab-${i.toString()}`}>el.props.title</Tab>
       ))}
     </TabList>
-    {this.props.children.map((el, i) => <TabPanel>{el.props.title}</TabPanel>)}
+    {this.props.children.map((el, i) => (
+      <TabPanel>{el.props.title}</TabPanel>
+    ))}
```

Comment:
Looks like you ran prettier or something like that in the entire code base. This shouldn't be part of this PR.

Reply:
Yeah, I ran the linter script as specified in the `package.json` (i.e. `yarn lint::all::fix`) before pushing the commits. It automatically made a lot of changes in multiple files, files I hadn't even touched. But I'm sure none of the codebase was affected.